src/app/systelab-login/login.component.html

```diff
@@ -2,39 +2,137 @@
+                <div class="slab-form_input--marginTop15 text-right" *ngIf="activeRecovery === 1">
+                    <a href="#" class="slab-text slab-text--link  " (click)="goRecovery()">{{ 'COMMON_FORGOT_PASSWORD' | translate | async }}</a>
+                </div>
+                <button class="w-100 btn btn-primary slab-form__submit mt-4" [disabled]="inputUserName.invalid ||  inputPassword.invalid"
+                    (click)="doLogin()">{{ 'COMMON_ENTER' | translate | async }}</button>
+
+
+                <div class="slab-form_input--marginTop25 text-center" *ngIf="activeSignUp === 1">
+                    <a href="#" class="slab-text slab-text--link  " (click)="goSignUp()">{{ 'COMMON_CREATE_NEW_ACCOUNT' | translate | async }}</a>
+                </div>
+
+
+                <div *ngIf="errorUserPwd" class="alert alert-danger text-center slab-form_input--marginTop25">
+                    <a>{{ 'ERROR_INCORRECT_USERNAME_OR_PASSWORD' | translate | async }}</a>
+                </div>
+
+            
+                <div *ngIf="isLoading" class="text-center pt-2">
+                    {{ 'COMMON_LOADING' | translate | async }}...
+                    <i class="icon-spinner spin"></i>
+                </div>
+                <div *ngIf="!isLoading" class="text-center pt-2">
                     &nbsp;
                 </div>
-                <div *ngIf="isLoading" class="text-center pt-2" >
+            </form>
+        </div>
+        <div class="d-flex slab-form align-content-center flex-wrap mt-4" *ngIf="typeForm === 'signup'">
+            <div class="mb-4 slab-text slab-text--large slab-text--muted">{{ 'COMMON_SESSION_SIGNUP' | translate | async }}</div>
+            <form novalidate class="w-100">
+
+                <input class="w-100 slab-form__input slab-form__input--topradius" [ngClass]="{'slab-form_input--error':  inputUserName.invalid && inputUserName.touched }"
+                    id="inputUserName" [(ngModel)]="userName" placeholder="{{ 'COMMON_USERNAME' | translate | async }}" #inputUserName="ngModel"
+                    name="inputUserName" ng-enter="doSignUp()" type="text" required autocomplete="off" maxlength="20" minlength="6">
+                <input type="password" [ngClass]="{'slab-form_input--error':  inputPassword.invalid && inputPassword.touched }" placeholder="{{ 'COMMON_PASSWORD' | translate | async }}"
+                    autocomplete="off" class="w-100 slab-form__input slab-form__input--bottomradius" id="inputPassword" #inputPassword="ngModel"
+                    name="inputPassword" ng-enter="doSignUp()" [(ngModel)]="password" [ngModelOptions]="{standalone: true}" pattern="((?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{4,20})"
+                    required maxlength="20" minlength="6" />
+
+                <div class="slab-form_input--marginTop15 text-center">
+                    <small class="slab-text slab-text--xsmall badge badge-warning">{{ 'COMMON_PASSWORD_STR' | translate | async }}</small>
```

Comment:
I suggest to follow the rule 50 + 10 lines. If not is a large template that nobody understands.